llvm/test/CodeGen/AArch64/sve-interleave_accesses4-load.ll

```diff
@@ -0,0 +1,80 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=aarch64--linux-gnu -mattr=+sve < %s | FileCheck %s
+
+%struct.xyzt = type { i32, i32, i32, i32 }
+
+define void @interleave(ptr noalias nocapture noundef writeonly %dst, ptr nocapture noundef readonly %a, <vscale x 4 x i32> %x) {
```

Comment:
Given the changes are at the IR level I think it best for the tests to use `opt` rather than `llc`. Also, it is better to test the two idioms (i.e. interleave4 and deinerleave4) in isolation.  If you look at `llvm/test/Transforms/InterleavedAccess/AArch64/sve-interleaved-accesses.ll` you'll see what I mean on both counts.